lib/config/rules/typescript.js

```diff
@@ -123,4 +119,66 @@ module.exports = {
   '@typescript-eslint/promise-function-async': 'off',
   // When adding two variables, operands must both be of type number or of type string
   '@typescript-eslint/restrict-plus-operands': 'error',
+  // Bans “// @ts-ignore” comments from being used
+  '@typescript-eslint/ban-ts-ignore': 'error',
+  // Enforce consistent brace style for blocks
+  '@typescript-eslint/brace-style': 'error',
+  // Enforces consistent usage of type assertions.
+  '@typescript-eslint/consistent-type-assertions': 'error',
+  // Consistent with type definition either interface or type
+  '@typescript-eslint/consistent-type-definitions': 'error',
+  // Require or disallow spacing between function identifiers and their invocations
+  '@typescript-eslint/func-call-spacing': 'error',
+  // Disallow empty functions
+  '@typescript-eslint/no-empty-function': 'error',
+  // Disallow unnecessary parentheses
+  '@typescript-eslint/no-extra-parens': 'error',
+  // Requires Promise-like values to be handled appropriately.
+  '@typescript-eslint/no-floating-promises': 'off',
+  // Disallows magic numbers
+  '@typescript-eslint/no-magic-numbers': 'off',
+  // Prevents conditionals where the type is always truthy or always falsy
+  '@typescript-eslint/no-unnecessary-condition': 'error',
+  // Warns if an explicitly specified type argument is the default for that type parameter
+  '@typescript-eslint/no-unnecessary-type-arguments': 'error',
+  // Prefer a ‘for-of’ loop over a standard ‘for’ loop if the index is only used to access the array being iterated
+  '@typescript-eslint/prefer-for-of': 'error',
+  // Requires that private members are marked as readonly if they're never modified outside of the constructor
+  '@typescript-eslint/prefer-readonly': 'error',
+  // Enforce the consistent use of either backticks, double, or single quotes
+  '@typescript-eslint/quotes': [
+    'error',
+    'single',
+    {avoidEscape: true, allowTemplateLiterals: true},
+  ],
+  // Enforce giving compare argument to Array#sort
+  '@typescript-eslint/require-array-sort-compare': 'error',
```

Comment:
I don't think I've ever dine this...

Reply:
Yeah same, I am going to turn it off.

Reply:
That said, I think there was only one violation in quilt. We don't `sort` too often.